agent/mitm/test/dns.test.ts

```diff
@@ -50,7 +50,7 @@ class CustomPlugin implements INetworkHooks {
   }
 
   onDnsConfiguration(settings) {
-    settings.dnsOverTlsConnection = Quad9;
+    settings.dnsOverTlsConnection = CloudFlare;
```

Comment:
This was the previous setting and they started blocking requests. Fine probably for now, but this might just cause new flakiness

Reply:
Yea quad was giving lots of problems and this seemed to work for now